jupyter_core/command.py

```diff
@@ -49,6 +49,7 @@ def list_subcommands():
     Returns a list of jupyter's subcommand names, without the `jupyter-` prefix.
     Nested children (e.g. jupyter-sub-subsub) are not included.
```

Comment:
This docstring seems to suggest that commands with further hyphens are treated as a hierarchical set of subcommands, so `jupyter-install-nbextension` would be invoked as `jupyter install nbextension`. But that means we can't have any subcommands with hyphens in the name. To my mind, `install-nbextension` is one thing, not a subcommand of install; if we're doing an extra level like that, I'd make it `jupyter nbextension install` (similar to kernelspecs).

@minrk ?